.github/ISSUE_TEMPLATE/new-cfe-template.yml

```diff
@@ -0,0 +1,78 @@
+name: 🚧 Code freeze exception request
+description: Request handling one or more pull requests(s) as code freeze exception(s).
+title: "[CFE]: "
+labels: ["code freeze exception"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this template to request changes to be included in a version of WooCommerce that is past its code freeze date.
+        **The request will be reviewed** and accepted or denied:
+        
+        * If accepted, a pull request targeting the appropriate release branch will be created.
+        * If rejected, a pull request targeting `trunk` will be created.
+        
+        In either case you are responsible for managing the pull request as usual (provide a description, assign reviewers, ensure that CI jobs pass...)
+  - type: input
+    id: release-number
+    attributes:
+      label: "Which release does this exception apply to?"
+      description: "Format: X.Y"
+      placeholder: "7.0"
+    validations:
+      required: true
+  - type: textarea
+    id: pr-urls
+    attributes:
+      label: "What PR implement the changes for which the exception is requested? (please do not enter multiple PRs)"
+      description: "Pull request URL"
+      placeholder: |
+        https://github.com/woocommerce/woocommerce/pull/1234
+    validations:
+      required: true
+  - type: textarea
+    id: why-needed
+    attributes:
+      label: "Why do these PRs need to break the code freeze?"
+      placeholder: "This is a revert of ... which introduced a bug that causes ..."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      id: consequence-if-not-included
+      label: "What is the consequence if this exception does not get included?"
+      placeholder: "The ... flow will be broken for ... users"
+    validations:
+      required: true
+  - type: textarea
+    id: plan-if-defects-discovered
+    attributes:
+      label: "What is the plan should defects to be discovered in these PRs ahead of final release?"
+      placeholder: "Reverting this PR and ... would be enough"
+    validations:
+      required: true
+  - type: textarea
+    id: how-to-communicate
+    attributes:
+      label: "How should this change be communicated in the release post on the [public developer blog](https://href.li/?https://developer.woo.com/blog/)?"
```

Comment:
Nit: while testing this template I noticed the md link is not working on the `label` attribute:

<img width="870" alt="Screenshot 2024-09-18 at 12 12 45" src="https://github.com/user-attachments/assets/44dbe13c-8e1f-46fe-b212-f1592d41a109">

Maybe we can add the link in the `description` field, it seems to work well there:

<img width="880" alt="Screenshot 2024-09-18 at 12 12 18" src="https://github.com/user-attachments/assets/f859b544-bd46-422d-be3a-2366543a5758">

Something similar to:

```md
- type: textarea
    id: how-to-communicate
    attributes:
      label: "How should this change be communicated in the release post on the public developer blog:"
      description: "See the blog at [https://developer.woo.com/blog/](https://developer.woo.com/blog/)"
      placeholder: "There is no need to add new communication to the already planned one."
    validations:
      required: true

Reply:
Good catch, fixed it here: https://github.com/woocommerce/woocommerce/pull/51468/commits/f819e7942992342345b2aa838f717015683fe28a